src/filesystem/impls/filer/lib/CSSRewriter.js

```diff
@@ -28,6 +28,7 @@ define(function (require, exports, module) {
         function aggregate(content, callback) {
             var urls = [];
             var urlRegex = new RegExp('url\\([\\\'\\"]?([^\\\'\\"\\)]+)[\\\'\\"]?\\)', 'g');
+            var urlQuotesRegex = new RegExp(/[\\\'\\"]([^\\\'\\"\\)]+)[\\\'\\"]/, 'g');
```

Comment:
@humphd instead of making `urls` into an object, can we keep the current way we push url strings into the `urls` array and then just use the `filter` method to extract the unique elements like this -
`urls = urls.filter((v, i, a) => a.indexOf(v) === i);`